src/site/apt/building/cmake.apt

```diff
@@ -0,0 +1,104 @@
+~~ Licensed to the Apache Software Foundation (ASF) under one or more
+~~ contributor license agreements.  See the NOTICE file distributed with
+~~ this work for additional information regarding copyright ownership.
+~~ The ASF licenses this file to You under the Apache License, Version 2.0
+~~ (the "License"); you may not use this file except in compliance with
+~~ the License.  You may obtain a copy of the License at
+~~
+~~      http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing, software
+~~ distributed under the License is distributed on an "AS IS" BASIS,
+~~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~~ See the License for the specific language governing permissions and
+~~ limitations under the License.
+ ------
+ Building Apache log4cxx with CMake
+ ------
+ ------
+ ------
+
+* Quick start:
+
+  Building and testing log4cxx on a Unix platform with packaged APR and APR-Util.
+
+  Make sure cmake 3.13+, g++ and make are available, install or
+  build apr 1.x, apr-util 1.x, gzip and zip.
+
++------------+
+$ apt-get install build-essential libapr1-dev libaprutil1-dev gzip zip
+$ cd apache-log4cxx-x.x.x
+$ mkdir build 
+$ cd build
+$ ccmake ..
+$ make
+$ sudo make install
++------------+
+
+* ccmake options
+*------------------------+---------------------------------------------------------------------------------------------+
```

Comment:
A newline is necessary BEFORE the table or it doesn't render properly and is instead interpreted as additional text for the headline `ccmake options` .

![Clipboard02](https://user-images.githubusercontent.com/6223655/74109084-b9017900-4b80-11ea-8115-4cb9728217a4.png)

Reply:
The socketservertestcase fails on Windows for me too. I have not investigated the cause.
It also (sometimes) fails on Linux (Ubuntu) when run from Maven. It always seem to succeed on Linux when I run ctest from the command prompt.

Reply:
Six test cases (including socketservertestcase) require the 'sed' program to pass.

I have now excluded those when the 'sed' program is not available.